packages/twenty-server/src/modules/contact-creation-manager/services/create-company.service.ts

```diff
@@ -157,10 +153,10 @@ export class CreateCompanyService {
     };
   }
 
-  private createCompanyMap(companies: CompanyWorkspaceEntity[]) {
+  private createCompanyMap(companies: DeepPartial<CompanyWorkspaceEntity>[]) {
     return companies.reduce(
       (acc, company) => {
-        if (!company.domainName) {
+        if (!company.domainName?.primaryLinkUrl || !company.id) {
```

Comment:
curious why this came now? wouldn't this fail two line laters if domainName is null?